README.rst

```diff
@@ -314,8 +314,8 @@ then you can set the variable to use a relative path. For example:
 **Customising pip install invocation**
 
 By default `mkvenv` will install setup.py via pip in editable (i.e. development) mode. See
-`here <http://codumentary.blogspot.com/2014/11/python-tip-of-year-pip-install-editable.html/>`__
-and `here <https://pip.pypa.io/en/stable/reference/pip_install/#editable-installs/>`__ for
+`here <http://codumentary.blogspot.com/2014/11/python-tip-of-year-pip-install-editable.html>`__
```

Comment:
actually would you mind removing the codumentary link? We should only be linking to official docs where possible

Reply:
Sure :)